hardware/Dummy.cpp

```diff
@@ -394,10 +396,13 @@ namespace http {
 			{
 				root["status"] = "OK";
 				root["title"] = "CreateVirtualSensor";
+				std::stringstream ss;
+				ss << vs_idx;
+				root["idx"] = ss.str().c_str();
 			}
 			if (DeviceRowIdx != -1)
 			{
-				m_sql.safe_query("UPDATE DeviceStatus SET Name='%q', Used=1 WHERE (ID==%" PRIu64 ")", ssensorname.c_str(), DeviceRowIdx);
+				m_sql.safe_query("UPDATE DeviceStatus SET Name='%q', Options='%q', Used=1 WHERE (ID==%" PRIu64 ")", ssensorname.c_str(), soptions.c_str(), DeviceRowIdx);
```

Comment:
Ha right, 
I was not confident about this part. The thing is that there were no way through JSON to modify the Options.
Override of the default Temperature sensor options was problematic (LA Crosse TX... by default).

I think you would prefer a separated JSON call to set the Options, right ?

You can remove this from the pull request for now. I'll setup a separate way to cope with it.

Many thanks for your review so far.